bf_opencore/tests/test_asset.py

```
@@ -26,15 +29,15 @@ def test_get_empty_assets(auth_client):
 
 def test_create_asset(auth_client):
     """Creating an empty asset makes it available via the API."""
-    dummy_asset = bf_mod.Asset.objects.create()
+    dummy_asset = models.Asset.objects.create()
```

Ruff (0.15.1)

[error] Local variable `dummy_asset` is assigned to but never used

Remove assignment to unused variable `dummy_asset`

(F841)